lib/purescript-cst/package.yaml

```diff
@@ -0,0 +1,77 @@
+name: purescript-cst
+version: '0.13.6'
+synopsis: PureScript Programming Language Concrete Syntax Tree
+description: The surface syntax of the PureScript Programming Language.
+category: Language
+author: Phil Freeman <paf31@cantab.net>
+maintainer: >
+  Gary Burgess <gary.burgess@gmail.com>,
+  Hardy Jones <jones3.hardy@gmail.com>,
+  Harry Garrood <harry@garrood.me>,
+  Christoph Hegemann <christoph.hegemann1337@gmail.com>,
+  Liam Goodacre <goodacre.liam@gmail.com>,
+  Nathan Faubion <nathan@n-son.com>
+copyright: (c) 2013-17 Phil Freeman, (c) 2014-19 Gary Burgess, (c) other contributors (see CONTRIBUTORS.md)
+license: BSD3
+github: purescript/purescript
+homepage: http://www.purescript.org/
+extra-source-files:
+  - tests/purs/layout/*.purs
+  - README.md
+dependencies:
+  - array
+  - base >=4.11 && <4.13
+  - containers
+  - dlist
+  - purescript-ast
+  - scientific >=0.3.4.9 && <0.4
+  - semigroups >=0.16.2 && <0.19
+  - text
+build-tools:
+  - happy ==1.19.9
+
+library:
+  source-dirs: src
+  ghc-options: -Wall -O2
+  default-extensions:
+    - ConstraintKinds
+    - DataKinds
+    - DeriveFunctor
+    - DeriveFoldable
+    - DeriveTraversable
+    - DeriveGeneric
+    - DerivingStrategies
+    - EmptyDataDecls
+    - FlexibleContexts
+    - KindSignatures
+    - LambdaCase
+    - MultiParamTypeClasses
+    - NoImplicitPrelude
+    - PatternGuards
+    - PatternSynonyms
+    - RankNTypes
+    - RecordWildCards
+    - OverloadedStrings
+    - ScopedTypeVariables
+    - TupleSections
+    - ViewPatterns
```

Comment:
Similar here about `default-extensions`. Marking this so we don't lose track of it.